website/grading.rst

```diff
@@ -3,92 +3,128 @@
 Grading
 =======
 
-.. warning::
+Like CMSC 14200, we will be using a form of grading known as *specifications grading*.
+The goal of specifications grading is to help students focus on their mastery of the
+material and to identify areas for improvement as the quarter progresses. You will
+get the most out of this scheme if you focus on skills, not on scores.
```

Comment:
Given that you're keeping this as a paragraph, could be good to either fully bold the last sentence, or at least bold the "skills" part of "skills, not on scores"